api/package.json

```diff
@@ -41,32 +41,28 @@
     "@sentry/node": "^5.5.0",
     "@slack/client": "^4.5.0",
     "@types/agenda": "1.0.3",
+    "@types/async-retry": "^1.4.1",
     "@types/body-parser": "^1.17.0",
+    "@types/cookie": "^0.3.1",
     "@types/cookie-parser": "^1.4.1",
     "@types/debug": "^4.1.5",
+    "@types/dependency-tree": "^6.1.0",
     "@types/dotenv": "^4.0.3",
     "@types/express": "^4.16.0",
     "@types/get-stdin": "^5.0.1",
     "@types/glob": "^5.0.35",
     "@types/jest": "^23.3.1",
     "@types/json5": "^0.0.30",
     "@types/jsonwebtoken": "^7.2.8",
-    "@types/lodash": "^4.14.116",
```

Comment:
Did Lodash start shipping its own types>?

Reply:
Yep, it probably won't build without them

Reply:
I restored the types packages, but I still needed to remove the `@types/mongoose`, otherwise it raises this error when running:
```
yarn run v1.22.5
$ node out/index.js
internal/modules/cjs/loader.js:638
    throw err;
    ^

Error: Cannot find module '/app/out/index.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)
error Command failed with exit code 1.
```
Is it ok to remove this package in this case or should I do something different?

Reply:
It must be failing to build entirely, you need to check the building phase